src/test/java/uk/gov/companieshouse/overseasentitiesapi/mocks/Mocks.java

```diff
@@ -71,4 +74,14 @@ private static List<BeneficialOwnerCorporateDto> buildBeneficialOwnersCorporateI
         beneficialOwnersCorporateInFiling.add(beneficialOwnerCorporateDto);
         return beneficialOwnersCorporateInFiling;
     }
+
+    private static List<ManagingOfficerIndividualDto> buildManagingofficersIndividualInFiling() {
```

Comment:
Type-o in buildManagingofficersIndividualInFiling method name
Officers should start with capital O